tests/integration/test_events_health_and_daemon_pid.py

```
@@ -429,3 +429,212 @@ def test_is_running_false_for_dead_pid(self, tmp_path: Path) -> None:
+
+        if os.name == "nt" or not hasattr(os, "chown"):
+            pytest.skip("os.chown is Unix-only")
+        from daemon.pid import PidFileManager
+
+        pid_file = tmp_path / "owned.pid"
+        pid_file.write_text("0")
+        stat = pid_file.stat()
+
+        calls: list[tuple[str, int, int]] = []
+
+        def fake_chown(path, uid, gid):  # type: ignore[no-untyped-def]
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `fake_chown`

Add return type annotation: `Never`

(ANN202)